pkgs/development/tools/build-managers/bob/default.nix

```diff
@@ -11,9 +17,18 @@ buildGoModule rec {
     hash = "sha256-zmWfOLBb+GWw9v6LdCC7/WaP1Wz7UipPwqkmI1+rG8Q=";
   };
 
+  patches = [
+    # Fix vulnerable dependencies
+    # Backport of https://github.com/benchkram/bob/pull/387
+    (fetchpatch {
+      url = "https://github.com/katexochen/bob/commit/5020e6fafbfbcb1b3add5d936886423ce882793d.patch";
```

Comment:
Can you please put the patch directly in nixpkgs source? This would make sure the package builds even if you end up removing your fork.

Reply:
You could also use `https://github.com/benchkram/bob/commit/5020e6fafbfbcb1b3add5d936886423ce882793d.patch`. The commit is indirectly included in the main repo because of the PR.